uint8 rate
---
bool success   # indicate successful run of triggered service
string message # informational, e.g. for error messages